contracts/src/chainlink/solidity/emergency/StarkNetValidator.sol

```diff
@@ -6,34 +6,39 @@ import "@chainlink/contracts/src/v0.8/interfaces/TypeAndVersionInterface.sol";
 import "@chainlink/contracts/src/v0.8/interfaces/AccessControllerInterface.sol";
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 import "@chainlink/contracts/src/v0.8/SimpleWriteAccessController.sol";
-
-import "@chainlink/contracts/src/v0.8/dev/vendor/openzeppelin-solidity/v4.3.1/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
 
 import "../../../../vendor/starkware-libs/starkgate-contracts-solidity-v0.8/src/starkware/starknet/solidity/IStarknetMessaging.sol";
 
-/**
- * @title StarkNetValidator - makes cross chain call to update the Sequencer Uptime Feed on L2
- */
+/// @title StarkNetValidator - makes cross chain calls to update the Sequencer Uptime Feed on L2
 contract StarkNetValidator is TypeAndVersionInterface, AggregatorValidatorInterface, SimpleWriteAccessController {
   int256 private constant ANSWER_SEQ_OFFLINE = 1;
-  // Selector hardcoded because StarkNet hash function is not available in this environment
+  // The selector is the starknet_keccak hash of the function name 'update_status'.
+  // Notice: hardcoded b/c starknet_keccak is not available in this environment.
   uint256 constant STARK_SELECTOR_UPDATE_STATUS =
```

Comment:
We could actually compute this in-place (easier to understand and update) by using `keccak256` and cleaning the high 6 bits: https://docs.starknet.io/docs/Hashing/hash-functions#starknet-keccak

> StarkNet keccak, usually denoted by sn\_keccaksn_keccak, is defined as the first 250 bits of the Keccak256 hash (this is just Keccak256 augmented in order to fit into a field element).

Reply:
Wait I'm a bit confused.  I had left a comment in an earlier PR about hardcoding this and was told that we have to hardcode it here.  Is this no longer the case?

Reply:
We can either hard code, or compute it by slightly modifying the output of Solidity `keccak256`.

Hardcode is fine, computing it is better - some small cost on construction, but more readable core easier to support. Not a big priority, just an improvement proposal for when we solve the main open items.